projects/canopy/src/lib/pagination/pagination.component.html

```diff
@@ -0,0 +1,47 @@
+<nav
+  role="navigation"
+  aria-label="Pagination Navigation"
+  *ngIf="pages.length > 1"
+  [class]="'lg-pagination'"
+>
+  <div>
+    <ul>
+      <li>
+        <button
+          type="button"
+          class="lg-pagination__button"
+          aria-label="Go to previous page"
+          lgMarginRight="xxs"
+          (click)="previous()"
+          [disabled]="currentPage === 1"
+        >
+          <lg-icon name="chevron-left"></lg-icon>
+        </button>
+      </li>
+      <li *ngFor="let page of pages">
+        <button
+          type="button"
+          class="lg-pagination__button"
+          lgMarginRight="xxs"
+          [attr.aria-label]="'Go to page ' + page"
+          [attr.aria-current]="currentPage === page ? 'true' : null"
+          [class.lg-pagination__button-active]="currentPage === page"
+          (click)="goTo(page)"
+        >
+          {{ page }}
+        </button>
+      </li>
+      <li>
+        <button
+          type="button"
+          class="lg-pagination__button"
+          aria-label="Go to next page"
+          (click)="next()"
+          [disabled]="currentPage === numPages"
+        >
+          <lg-icon name="chevron-right"></lg-icon>
+        </button>
+      </li>
+    </ul>
```

Comment:
Looks like we are missing the text `Showing x-y of z results` from the design.

Reply:
This was deliberate (on my part) as for now, I only wanted to implement the this is only the most basic version on the control we need as the rest of the design is yet to be signed off. I can add it in but we will need to agree how to display it (either automatically place it to the right or below the pagination, depending on the screen size, if via an input such as  `@Input() labelPlacement = 'none', 'right', 'below', 'auto'`

Reply:
From the designs, I think it's always positioned to the right apart from when it doesn't fit and it goes below so I don't think we need a new input?
I'd leave out the `Got to value` for now.

Reply:
I've added the label with auto positioning

![Screenshot 2023-02-27 at 15 48 46](https://user-images.githubusercontent.com/22469863/221611764-2555481e-dbe2-42b1-9cde-3af3ef8f2bf2.png)
![Screenshot 2023-02-27 at 15 48 28](https://user-images.githubusercontent.com/22469863/221611731-4c62512b-ab87-4c88-87f4-2463866a09ac.png)